.azure-pipelines.yml

```diff
@@ -80,6 +80,16 @@ stages:
       iosVariableGroup: 'ApplicationTemplate.Distribution.Internal.iOS'
       BannerVersionNameText: "STAGING"
 
+- stage: Firebase_App_Distribution
```

Comment:
You shouldn't have a new stage, you need to change the existing one to replace AppCenter in favor of Firebase app distribution.

Reply:
im currently testing I was planning to remove them after